Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -176,6 +180,18 @@ public bool TryGetEntry(OperationContext context, ShortHash hash, out ContentLoc
         /// </summary>
         public abstract IEnumerable<ShortHash> EnumerateSortedKeys(CancellationToken token);
 
+        /// <summary>
+        /// Enumeration filter used by <see cref="ContentLocationDatabase.EnumerateEntriesWithSortedKeys"/> to filter out entries by raw value from a database.
+        /// </summary>
+        public delegate bool EnumerationFilter(byte[] value);

```

Comment:
>EnumerationFilter [](start = 29, length = 17)

Does this need a dedicated type?

Reply:
I think so, because I would like to emphesize what is 'byte[]' argument is.

---
In reply to: [276376653](https://github.com/Microsoft/BuildXL/pull/150#discussion_r276376653) [](ancestors = 276376653)